docs/src/snap/howto/networking/dualstack.md

```diff
@@ -138,6 +138,7 @@ cluster bootstrap process. The key configuration parameters are:
 When setting up dual-stack networking, it is important to consider the
 limitations regarding CIDR size:
 
-- **/64 is too large for the Service CIDR**: Using a `/64` CIDR for services
+- **/108 is the maximum size for the Service CIDR**
+Using a smaller valuethan `/108` for service CIDRs
```

Comment:
Is this a limitation with upstream? It might be worth to link to an upstream issue or documentation page.

Reply:
Idk, seems controversial: https://github.com/kubernetes/kubernetes/pull/101457

Reply:
You can link directly to the tests: https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/options/validation_test.go#L435

Reply:
Ahh nice find! Will do.